hooks/conan-center.py

```diff
@@ -651,6 +654,45 @@ def test(out):
                     out.error("The file '{}' uses CRLF. Please, replace by LF."
                               .format(filename))
 
+    @run_test("KB-H061", output)
+    def test(out):
+        Location = collections.namedtuple("Location", ("line", "column"))
+        BuildInfo = collections.namedtuple("BuildInfo", ("loc", "what", "func"))
+        class BuildInfoVisitor(ast.NodeVisitor):
+            INVALID_METHODS = (
+                "config_options",
+                "configure",
+                "build_requirements",
+                "requirements",
+                "source",
+                "validate",
```

Comment:
```suggestion
                "requirements",
                "validate",
                "package_id",
                "build_requirements",
                "source",
```

And `system_requirements()`?

Why `source()`?

Reply:
these are legit cases for `os_info`
right now we only block problematic methods for CI, for package id computation on linux node

Reply:
Code in `source` should be independent of `settings`/`options`, and thus also `os_info`. It should be possible to run `source` once, archive the `source` and transport it to whatever machine you desire.

Adding the check to `system_requirements` will cause problems in some packages.
System requirements do not alter any package id calculation, I think.

Reply:
ok, but package_id() is still missing.